fontforgeexe/startui.c

```diff
@@ -463,10 +462,16 @@ static pascal OSErr OpenDocumentsAE( const AppleEvent * theAppleEvent,
                          typeAEList, &docList);
     err = AECountItems(&docList, &count);
     for(index = 1; index <= count; index++) {
-        err = AEGetNthPtr(&docList, index, typeFSRef,
-                        NULL, NULL, &theFSRef,
-                        sizeof(theFSRef), NULL);// 4
-	err = FSRefMakePath(&theFSRef,(unsigned char *) buffer,sizeof(buffer));
+        AEDesc aDoc;
+        err = AEGetNthDesc(&docList, index, typeFileURL, NULL, &aDoc);
+        size_t bytecount = AEGetDescDataSize(&aDoc);
+        void *pathPtr = malloc(bytecount);
+        err = AEGetDescData(&aDoc, pathPtr, bytecount);
```

Comment:
seems weird that we store the error but do nothing with it

Reply:
That's how the code was originally written. That said, using the `continue` directive when there's an error should prevent errors.